iaso/tests/models/test_payments.py

```diff
@@ -72,12 +74,17 @@ class PotentialPaymentModelTestCase(TestCase):
 
     @classmethod
     def setUpTestData(cls):
-        user = User.objects.create(username="test_user")
+        cls.user = User.objects.create(username="test_user")
         cls.org_unit = m.OrgUnit.objects.create(name="Test OrgUnit")
-        change_request = m.OrgUnitChangeRequest.objects.create(org_unit=cls.org_unit, new_name="Foo")
-        cls.potential_payment = m.PotentialPayment.objects.create(user=user)
-        cls.potential_payment.change_requests.add(change_request)
+        cls.change_request = m.OrgUnitChangeRequest.objects.create(org_unit=cls.org_unit, new_name="Foo")
 
     def test_create(self):
+        self.potential_payment = m.PotentialPayment.objects.create(user=self.user)
+        self.potential_payment.change_requests.add(self.change_request)
         self.assertEqual(self.potential_payment.user.username, "test_user")
         self.assertEqual(self.potential_payment.change_requests.first().new_name, "Foo")
```

Comment:
```suggestion
        potential_payment = m.PotentialPayment.objects.create(user=user)
        potential_payment.change_requests.add(change_request)
        assertEqual(potential_payment.user.username, "test_user")
        assertEqual(potential_payment.change_requests.first().new_name, "Foo")
```

We should use local variables in test methods because assigning instance attributes with `self` make these attributes available in all other test methods and this can pose subtle problems to debug.